packages/components/Pagination.tsx

```diff
@@ -17,17 +17,19 @@ import { SpacerRow } from "./spacer";
 
 // misc
 
-type PaginationProps = {
+interface PaginationProps {
```

Comment:
Why using `interface` instead of `type` ? (In general)

Reply:
in general:

https://blog.devgenius.io/interfaces-vs-types-in-typescript-7567ffe390cd

Reply:
interface can be extandable, ok thx !